src/data/fake_data.json

```diff
@@ -1,55 +1,55 @@
 {
-    "schedule": [
+  "schedule": [
```

Comment:
(same question as above)

Reply:
also just reformatting which is kinda weird